python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/platform.schema.yml

```diff
@@ -41,6 +41,52 @@ keys:
                   keys:
                     name:
                       type: str
+                    ingress:
+                      type: dict
+                      keys:
+                        priority_groups:
```

Comment:
you are not implementing the reserved / threshold at the ingress level

```
tps366...07:46:33(config-s-gmu-queue-BLAH)#ingress ?
  headroom        Specify the amount of headroom that should be reserved
  priority-group  Configure priority-group parameters
  reserved        Specify the amount of memory that should be reserved <<<<<<<<<<<
  resume          Specify the amount of memory that should be available before XON
  threshold       Specify the dynamic shared memory threshold <<<<<<<<<<<<<
```

is it on purpose? I think it is worth adding it here to have full flexibility

```
+platform trident mmu queue profile BLAH
+   ingress reserved 666
+   ingress priority-group 4 reserved 42
```

Reply:
Added!